### PR TITLE
Add Cell after current active cell

### LINKED ISSUE #10202: "Add Cell" must add new cell after active cell and not at the end
**Is your feature request related to a problem? Please describe.**

**Describe the solution or feature you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by rajeshka**
This PR fixes #10202

### CODE REVIEW

File: src/sql/workbench/contrib/notebook/browser/notebookActions.ts [hunk 1]
```diff
@@ -49,7 +50,17 @@ export class AddCellAction extends Action {
 	public run(context: INotebookEditor): Promise<boolean> {
 		return new Promise<boolean>((resolve, reject) => {
```
Comment (Charles-Gagnon) [inline src/sql/workbench/contrib/notebook/browser/notebookActions.ts:51]: While you're here - just make this function async and you can get rid of the wrapped promise and the try/catch

File: src/sql/workbench/contrib/notebook/browser/notebookActions.ts [hunk 2]
```diff
@@ -49,7 +50,17 @@ export class AddCellAction extends Action {
...
 			try {
-				context.addCell(this.cellType);
+				//Add Cell after current selected cell.
+				let notebookcomponent = context as NotebookComponent;
+				let id = notebookcomponent.activeCellId;
+				let index = undefined;
```
Comment (chlafreniere) [inline src/sql/workbench/contrib/notebook/browser/notebookActions.ts:56]: If context or context.cells or id is null/undefined, then index will still be undefined. may want to set it to a real value initially.

File: src/sql/workbench/contrib/notebook/browser/notebookActions.ts (hunk 1 above)
Comment (aaomidi) [reply src/sql/workbench/contrib/notebook/browser/notebookActions.ts:51]: Definitely this

Comment (rajeshka) [reply src/sql/workbench/contrib/notebook/browser/notebookActions.ts]: updated

File: src/sql/workbench/contrib/notebook/browser/notebookActions.ts (hunk 2 above)
Comment (rajeshka) [reply src/sql/workbench/contrib/notebook/browser/notebookActions.ts]: Done

**Comment by rajeshka** [comment]
> @rajeshka You have a broken test FYI

working on fixing it

**Pull request merged**